ghost/core/core/server/api/endpoints/session.js

```diff
@@ -33,6 +34,8 @@ const controller = {
         return models.User.getByEmail(object.username).then((user) => {
             if (user && !user.hasLoggedIn()) {
                 skipVerification = true;
+            } else if (frame.data.key === 'password_reset') {
```

Comment:
Should make this simpler too -- instead of caring about a string value, just saying `} else if (frame.data.skipEmailVerification) {`